CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 4.0
+
+- `.void()` now only accepts undefined, not null.
```

Comment:
(Is this considered breaking enough to warrant a bump to 4.0?)

Reply:
Nope